packages/openapi-v3/src/api-decorator.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ControllerSpec} from './';
```

Comment:
import from the file itself please

Reply:
Any reason or is it a documented convention to export it from the file itself?
For the ones exported in `src/index.ts`, I am used to import from root level

Reply:
Importing from `index.ts` can cause a circular dependency for some reason

Reply:
oh! that's important. changing all of them